cmd/alias-remove.go

```diff
@@ -61,28 +63,31 @@ func checkConfigHostRemoveSyntax(ctx *cli.Context) {
 	}
 }
 
-// mainConfigHost is the handle for "mc config host rm" command.
-func mainConfigHostRemove(ctx *cli.Context) error {
-	checkConfigHostRemoveSyntax(ctx)
+// mainAliasRemove is the handle for "mc alias rm" command.
+func mainAliasRemove(ctx *cli.Context, deprecated bool) error {
+	checkAliasRemoveSyntax(ctx)
 
-	console.SetColor("HostMessage", color.New(color.FgGreen))
+	console.SetColor("AliasMessage", color.New(color.FgGreen))
 
 	args := ctx.Args()
 	alias := args.Get(0)
-	removeHost(alias) // Remove a host.
+
+	aliasMsg := removeAlias(alias) // Remove an alias
+	aliasMsg.op = "remove"
```

Comment:
is there a reason for not setting `aliasMsg.op` inside `removeAlias` function?

Reply:
> is there a reason for not setting `aliasMsg.op` inside `removeAlias` function?

Setting the operation name outside is actually only needed in alias-set.go, because I renamed `add` to `set` and I wanted that `mc config host add` to return 'json.op = add' and `mc alias set` to return 'json.op = set'.

This is not a big deal for me. I can change it if you want.